static/locales/en-US.yaml

```diff
@@ -880,7 +880,7 @@ Tooltips:
     Hide Channels: Enter a channel ID to hide all videos, playlists and the channel itself from appearing in search, trending, most popular and recommended.
        The channel ID entered must be a complete match and is case sensitive.
     Hide Subscriptions Live: 'This setting is overridden by the app-wide "{appWideSetting}" setting, in the "{subsection}" section of the "{settingsSection}"'
-    Hide Videos and Playlists Containing Text: Enter a word, word fragment, or phrase (case insensitive) to hide all videos & playlists with titles containing it throughout all of FreeTube.
+    Hide Videos and Playlists Containing Text: Enter a word, word fragment, or phrase (case insensitive) to hide all videos & playlists with titles containing it throughout all of FreeTube, excluding only History and videos inside of playlists.
```

Comment:
```suggestion
Hide Videos and Playlists Containing Text: Enter a word, word fragment, or phrase (case insensitive) to hide all videos & playlists with titles containing it throughout all of FreeTube, excluding only your Playlists, History and videos inside of playlists.
```

Reply:
@jasonhenriquez

Reply:
Went with this but just moved the order: `Enter a word, word fragment, or phrase (case insensitive) to hide all videos & playlists with titles containing it throughout all of FreeTube, excluding only History, Your Playlists, and videos inside of playlists.`